charts/stable/traefik/templates/_args.tpl

```diff
@@ -142,9 +142,9 @@ args:
   {{- end }}
   {{- end }}
   {{/* theme.park */}}
-  {{- if .Values.themePark}}
+  {{- if .Values.middlewares.enableThemePark}}
   - "--experimental.plugins.traefik-themepark.modulename=github.com/packruler/traefik-themepark"
-  - "--experimental.plugins.traefik-themepark.version={{ .Values.themeParkVersion }}"
+  - "--experimental.plugins.traefik-themepark.version={{ .Values.middlewares.themeParkVersion }}"
```

Comment:
Update those aswell :)

Reply:
Good catch. I think the middlewares part is still required based on questions.yml and values.yaml structure

Reply:
Correct!